src/Validator/ConstraintsSetFactory/Field/JsonFactory.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+declare(strict_types=1);
+
+namespace AssoConnect\ValidatorBundle\Validator\ConstraintsSetFactory\Field;
+
+use Doctrine\DBAL\Types\Types;
+
+class JsonFactory implements FieldConstraintsSetFactoryInterface
+{
+    public function supports(string $type): bool
+    {
+        return Types::JSON === $type;
+    }
+
+    public function getConstraints(array $fieldMapping): array
+    {
+        // TODO: implement JSON validation?
+        return [];
```

Comment:
Symfony provide a [json constraint](https://symfony.com/doc/current/reference/constraints/Json.html) since symfony 4.3. Do you think it can be implemented here? Maybe it can introduce a braking change but it could improve validator.